docs/chapters/testing.tex

```diff
@@ -234,7 +234,7 @@ \subsection{Test 3: modify settings}
 \item Verify that the impacted length becomes \SI{65}{\metre}.
 \item Set the minimum flow-carrying discharge to \SI{2100}{\metre\cubed\per\second}
 \item Verify that the impacted length reduces to \SI{8}{\metre}, and that the entry boxes for all discharges less or equal to 2100 are greyed-out.
-\item Select from the folder \file{tests/c01 - GendtseWaardNevengeul} included in the \dfmi source code repository, the files \file{reference-Q1\_map.nc} and \file{measure-Q1\_map.nc} for the \SI{2500.0}{\metre\cubed\per\second} condition, and \file{reference-Q2\_map.nc} and \file{measure-Q2\_map.nc} for the \SI{3200.0}{\metre\cubed\per\second} condition as shown in \autoref{fig:test3.png}.
+\item Select from the folder \file{tests/c01 - GendtseWaardNevengeul} included in the \dfmi source code repository, the files \file{reference-Q1\_map.nc} and \file{intervention-Q1\_map.nc} for the \SI{2500.0}{\metre\cubed\per\second} condition, and \file{reference-Q2\_map.nc} and \file{intervention-Q2\_map.nc} for the \SI{3200.0}{\metre\cubed\per\second} condition as shown in \autoref{fig:test3.png}.
```

Comment:
bestaat intervention-Q1,Q2,Q3_map.nc ?

Reply:
lets have a look together

Reply:
@MRVermeulenDeltares When updating the the naming of these files also update related images when the references in documentation are updated.